module-1/task-1/index.js

```diff
@@ -0,0 +1,19 @@
+const readline = require('readline');
+
+const processLineByLine = () => {
+  const rl = readline.createInterface({
```

Comment:
Creating an instance of readline could be done outside of a running handler function so that it could be reused in more than one handler.
In your case, calling processLineByLine is redundant since Node.js executes the whole file anyway.
In a typical JS program the purpose of creating function is to reuse parts of the logic or call the function recursively.
In your case, calling processLineByLine will recreate the readline interface and reassign event listeners which could potentially lead to memory leaks.

Reply:
Fixed, thanks!

Reply:
Fixed, thanks!